randovania/games/fusion/logic_database/Sector 4 AQA.txt

```diff
@@ -1102,7 +1107,7 @@ Extra - room_id: [29]
   > Door to Entrance Save Room
       Trivial
 
-> Recharge Station; Heals? True
+> Recharge Terminal; Heals? True
```

Comment:
Why are these being renamed? IIRC these have in game test that says station

Reply:
These are not renamed. I mentioned earlier that all nav rooms and recharge room use `Terminal`. All the other sectors work that way too